src/main/java/essenmakanan/command/AddRecipeCommand.java

```diff
@@ -95,16 +95,21 @@ public void addWithTitleStepsTags() {
     private static void addSteps(String step, RecipeStepList recipeStepList, Tag tag) {
         String[] allSteps = step.trim().split("s/");
         for (String eachStep : allSteps) {
-            if (eachStep.length()>0) {
+            if (eachStep.length()>0 && eachStep.contains("d/")) {
```

Comment:
I think `eachStep.length()>0` should be `eachStep.length() > 0` for easier readability.